community-operators/aqua/aqua-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,452 @@
+            "name": "aqua"
+          },
+          "spec": {
+            "requirements": false,
+            "serviceAccount": "aqua-sa",
+            "deploy": {
+              "replicas": 1,
+              "service": "ClusterIP",
+              "image": {
+                "repository": "scanner",
+                "registry": "registry.aquasec.com",
+                "tag": "4.0",
+                "pullPolicy": "IfNotPresent"
+              }
+            },
+            "login": {
+              "username": "administrator",
+              "password": "Password1",
+              "host": "http://aqua-server-svc:8080"
+            },
+            "openshift": true
+          }
+        },
+        {
+          "apiVersion": "operator.aquasec.com/v1alpha1",
+          "kind": "AquaServer",
+          "metadata": {
+            "name": "aqua"
+          },
+          "spec": {
+            "requirements": false,
+            "add": false,
+            "serviceAccount": "aqua-sa",
+            "dbSecretName": "aqua-database-password",
+            "dbSecretKey": "db-password",
+            "rbac": {
+              "enable": true,
+              "openshift": true,
+              "privileged": true
+            },
+            "deploy": {
+              "replicas": 1,
+              "service": "LoadBalancer",
+              "image": {
+                "repository": "console",
+                "registry": "registry.aquasec.com",
+                "tag": "3.5",
+                "pullPolicy": "IfNotPresent"
+              }
+            },
+            "aquaDb": "aqua-database",
+            "adminPassword": "Password1",
+            "licenseToken": null,
+            "clusterMode": true,
+            "ssl": false,
+            "auditSsl": false,
+            "dockerless": false,
+            "openshift": true
+          }
+        }
+      ]
+spec:
+  displayName: Aqua Security Operator
+  version: 0.0.1
+  description: |-
+    The Aqua Security Operator runs within a Openshift cluster and provides a means to deploy and manage Aqua Security cluster and components as
+    * Server (Console)
+    * Gateway
+    * Database (Not recommended for production environments)
+    * Enforcer (Agent)
+    * Scanner CLI
+    * CSP (Package of Server, Gateway and Database)
+
+    Use the aqua-operator to - 
+    * Deploy Aqua Security components on Openshift
+    * Scale up Aqua Security components with extra replicas
+    * Assign metadata tags to Aqua Security components
+    * Automatic scale to the Aqua Scanner CLI by the count of the Scanning Queue.
+
+    ## After You Install
+    Once the operator is installed in the cluster, you now can use the crds to install aqua cluster and components.
+
+    ## Before You Begin Using the Operator CRDs
+    you need to create - 
+      * Service account with pull permissions from aqua docker registry
+      * Secret for docker registry 
+      * Secret for database
+
+    ```
+    oc create serviceaccount aqua-sa -n aqua
```

Comment:
Service accounts can be added to this CSV and created by OLM with the requested policies below, by adding an entry to `clusterPermissions` and creating securitycontextconstraint permissions. This has been done in this CSV, as a guide: https://github.com/operator-framework/community-operators/blob/master/community-operators/oneagent/dynatrace-monitoring.v0.2.0.clusterserviceversion.yaml#L203
The only manual step for OpenShift could be the secret creation.

Reply:
Since this step isn't needed for kubernetes, this should only be added to the community-operators CSV

Reply:
@SamiSousa 
Thanks Updated